xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/documentTags.vm

```diff
@@ -88,14 +88,14 @@
 ##
 #macro(displayAddForm)
 <form action="$doc.getURL('view', "xpage=documentTags&amp;xaction=add&amp;xredirect=${xredirect}")" method="post" class="tag-add-form">
-  <div>
-      ## CSRF prevention
-    <div class="hidden"><input type="hidden" name="form_token" value="$!{services.csrf.getToken()}" /></div>
-    <label for="tag">$services.localization.render('core.tags.add.label')<br/>
-      <input class="input-tag" type="text" id="tag" name="tag" autocomplete="off"/></label><br/>
-    <span class="buttonwrapper"><input class="button button-add-tag" type="submit" value="$services.localization.render('core.tags.add.submit')"/></span>
-    <span class="buttonwrapper"><a class="button button-add-tag-cancel" href="$doc.getURL('view', "#${tagsId}")">$services.localization.render('core.tags.add.cancel')</a></span>
-  </div>
+    <div>
+        ## CSRF prevention
+        <div class="hidden"><input type="hidden" name="form_token" value="$!{services.csrf.getToken()}" /></div>
+        <label for="tag">$services.localization.render('core.tags.add.label')<br/>
+            <input class="input-tag" type="text" id="tag" name="tag" autocomplete="off"/></label><br/>
+        <span class="buttonwrapper"><input class="button button-add-tag" type="submit" value="$services.localization.render('core.tags.add.submit')"/></span>
+        <span class="buttonwrapper"><a class="button button-add-tag-cancel" href="$doc.getURL('view', "#${tagsId}")">$services.localization.render('core.tags.add.cancel')</a></span>
+    </div>
```

Comment:
Why did you feel the need to change the existing indentation from 2 spaces (that we use on html and velocity code) to 4 spaces?

Please limit your changes to the issue at hand and respect the existing coding conventions. https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/

Reply:
Note that it does seem the original change was introduced in this commit https://github.com/xwiki/xwiki-platform/commit/fbe352330de2179e1e9f451b772cad9bbc8664a1#r48788975 and it seems that the file was already using bag indentation even before your change.

Reply:
> Why did you feel the need to change the existing indentation from 2 spaces (that we use on html and velocity code) to 4 spaces?
> 
> Please limit your changes to the issue at hand and respect the existing coding conventions. https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/

I apologize for this, I probably opened this file in two code editors and probably for these reasons the code was arranged differently. I made a new commit and I solved it